src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dayRenta = 40;
```

Comment:
The variable name 'dayRenta' is not descriptive enough and seems to be a typo. Consider renaming it to 'dailyRentalRate' for better clarity.